orangecontrib/associate/widgets/__init__.py

```diff
@@ -6,3 +6,18 @@
 # Background color for category background in menu
 # and widget icon background in workflow.
 BACKGROUND = "#cccc66"
+
+# Location of widget help files.
+WIDGET_HELP_PATH = (
```

Comment:
If this is something we use, please submit this, well documented, in a PR to https://github.com/biolab/orange3-example-addon.